cmd/fleet/metrics.go

```diff
@@ -83,6 +86,7 @@ func (rt *routeStats) Register(registry *monitoring.Registry) {
 }
 
 func init() {
+	metrics.SetupMetrics("fleet-server")
```

Comment:
This was required to be changed, because it cannot be called more than once per process.

If it is called more than once it will panic.